lib/utils.pm

```diff
@@ -1022,4 +1023,22 @@ sub svirt_host_basedir {
     return get_var('VIRSH_OPENQA_BASEDIR', '/var/lib');
 }
 
+sub prepare_ssh_localhost_key_login {
+    # in case localhost is already inside known_hosts
+    if (script_run('test -e ~/.ssh') == 0) {
+        assert_script_run('ls -l ~/.ssh');
```

Comment:
what is that for, for debugging only? Better just call code like this in a post_fail_hook when necessary